libraries/ESP8266WiFi/src/WiFiUdp.cpp

```diff
@@ -118,7 +118,7 @@ int WiFiUDP::available() {
     if (!result) {
         // yielding here will not make more data "available",
         // but it will prevent the system from going into WDT reset
-        optimistic_yield(1000);
+        optimistic_yield(100UL);
```

Comment:
(here and other spots)
In #6869 you'd suggested a happy medium of `10000` us.  I think it needs to be consistent here and elsewhere.  The `100` that's in these spots seems like it will always fire because the delay is so small, and because they're not "yield to the OS" but instead "don't WDT" it seems like we'd benefit from the longer offset (i.e. higher throughput).

Reply:
Also, can we be consistent on the `UL` or not `UL`?  I think in this PR 80% of the calls have `UL`, but IIRC none in #6869 have `UL`.